### PR TITLE
{bio}[GCC/8.2.0-2.31.1] TreeShrink v1.3.2 w/ Python 3.7.2

### PR DESCRIPTION
**Pull request by omula**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/TreeShrink/TreeShrink-1.3.2-GCC-8.2.0-2.31.1-Python-3.7.2.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'PythonPackage'
+
+name = 'TreeShrink'
+version = '1.3.2'
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/TreeShrink/TreeShrink-1.3.2-GCC-8.2.0-2.31.1-Python-3.7.2.eb]: There is no official 1.3.2 version of this yet.
So please push the developers to release a 1.3.2 version first.

Comment (omula) [reply easybuild/easyconfigs/t/TreeShrink/TreeShrink-1.3.2-GCC-8.2.0-2.31.1-Python-3.7.2.eb]: @akesandgren it seems they are not creating the releases in github but they are labeling them with commits.
They have even labeled a 1.3.3.
https://github.com/uym2/TreeShrink/commits/master

Shall I push developers to create those releases in github?

Comment (akesandgren) [reply easybuild/easyconfigs/t/TreeShrink/TreeShrink-1.3.2-GCC-8.2.0-2.31.1-Python-3.7.2.eb]: Yes, try to make them understand that they need to make an actual release, or a the very least a tag, although an actual release is much preferred. (A tag can be moved causing things to get out of sync)
Just having it named as v1.3.2 in a commit is not sufficient for people to be able to know what v1.3.2 really is.

**Comment by akesandgren** [comment]
It's also appreciated if you make test reports from the PR yourself.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/a83ce46c57618f97111118bb7d4ded7b for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @omula!

**Pull request merged**